### ISSUE TITLE
@Operation.parameters overriden by schema calculation based on type

### ISSUE THREAD

**Issue by antechrestos** [springdoc/springdoc-openapi#60]
As it was spotted in issue #13 , the following code

```kotlin
@GetMapping(value = ["/values/{trackerId}/data"])
    @Operation(description = "Get last data from a tracker",
            parameters = [
                Parameter(name = "trackerId", `in` = ParameterIn.PATH, required = true, schema = Schema(type = "string", example = "the-tracker-id")),
                Parameter(name = "start", `in` = ParameterIn.QUERY, required = false, schema = Schema(type = "string", format = "date-time", required = false, example = "1970-01-01T00:00:00.000Z")),
                Parameter(name = "end", `in` = ParameterIn.QUERY, required = false, schema = Schema(type = "string", format = "date-time", required = false, example = "1970-01-01T00:10:00.000Z")),
                Parameter(name = "limit", `in` = ParameterIn.QUERY, required = false, schema = Schema(type = "number", required = false, example = "10"))
            ],
            responses = [
                ApiResponse(responseCode = "200", content = [Content(array = ArraySchema(schema = Schema(implementation = ListDataResponse::class)))])
            ])
    fun list(@PathVariable trackerId: String,
                      @RequestParam(value = "start", required = false) start: Instant?,
                      @RequestParam(value = "end", required = false) end: Instant?,
                      @RequestParam(value = "limit", required = false) limit: Int?): Collection<ListDataResponse> = super.list(trackerId, start, end, limit)
``` 

leads to `start`and `end` query parameters resolved as `integer/int64` parameters instead of `string/date-time`

Seen in `1.1.26`

**Comment by springdoc** [comment]
Hi, 

 A fix for this issue is available on v1.1.28 of springdoc-openapi.
 You are invited to test it and give us your feedback.

**Comment by antechrestos** [comment]
@springdoc it works like a charm :ok_hand:
